{"name": "Unsuccessfully login with email: simeon.hhl.qa@gmail.com and password: Test#123", "status": "passed", "steps": [{"name": "Check for error", "status": "passed", "start": 1730460752314, "stop": 1730460752403}, {"name": "Check the error text", "status": "passed", "start": 1730460752403, "stop": 1730460752416}], "parameters": [{"name": "email", "value": "'simeon.hhl.qa@gmail.com'"}, {"name": "password", "value": "'Test#123'"}, {"name": "error", "value": "'There was a problem logging you in. Please try again.'"}, {"name": "driver", "value": "'edge'"}], "start": 1730460751390, "stop": 1730460752416, "uuid": "018bc097-1929-415c-a326-39fdee7169be", "historyId": "3bf7edab350b73c43d4821286022e547", "testCaseId": "c2811b1721d8231bbb0fb40778ae099c", "fullName": "Tests.test_Login.TestLogin#test_2", "labels": [{"name": "feature", "value": "Login"}, {"name": "severity", "value": "normal"}, {"name": "tag", "value": "flaky(reruns=3, reruns_delay=0.5, rerun_except='assert')"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "fv-az714-229"}, {"name": "thread", "value": "1843-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_Login"}]}